lib/taxjar/api/request.rb

```diff
@@ -4,7 +4,8 @@
 module Taxjar
   module API
     class Request
-      BASE_URL = 'https://api.taxjar.com'
+      DEFAULT_API_URL = 'https://api.taxjar.com'
+      SANDBOX_API_URL = 'https://api.sandbox.taxjar.com'
```

Comment:
this doesn't seem to be used anywhere, maybe it can be used to support an easy toggable flag? Or is it meant to just be a way for users to reference this in their own code?
```
client = Taxjar::Client.new(api_key: 'YOUR_SANDBOX_API_TOKEN', api_url: 'https://api.sandbox.taxjar.com')
->
client = Taxjar::Client.new(api_key: 'YOUR_SANDBOX_API_TOKEN', sandbox: true)
```

could be a way to reduce user error in setting the correct URL

Reply:
@slamorsi Exactly, you could use the constant in your own code similar to our PHP client:

```ruby
client = Taxjar::Client.new(api_key: 'YOUR_SANDBOX_API_TOKEN', api_url: Taxjar::API::Request::SANDBOX_API_URL)
```

PHP client:

```php
$client->setApiConfig('api_url', TaxJar\Client::SANDBOX_API_URL);
```

Considered a sandbox flag, but I figured `api_url` would be more flexible for future testing environments and API mocking tools.